browser/ui/views/tabs/brave_tab_context_menu_contents.cc

```diff
@@ -187,6 +187,8 @@ bool BraveTabContextMenuContents::IsBraveCommandIdEnabled(
     case BraveTabMenuModel::CommandBringAllTabsToThisWindow: {
       return true;
     }
+    case BraveTabMenuModel::CommandCloseDuplicateTabs:
+      return true;
```

Comment:
Optional: I wonder if we should only enable it if there are duplicate tabs? Not really important, tbh

Reply:
https://github.com/brave/brave-core/pull/22041/commits/56bab6f5a9bc9908b2de8786da77a7d4023367c8